### ISSUE TITLE
Provide option to use fixed canvas instead of infinite

### ISSUE THREAD

**Issue by Kruptein** [Kruptein/PlanarAlly#93]
Add DM option to choose between the current infinite canvas approach or a fixed X by Y canvas

**Comment by schemen** [comment]
I got an idea here: How about switching the canvas to some % bigger than the map (Dynamic Canvas)?
You select a map let's say 1000x2000 pixel, now the canvas gets set around that map, maybe 50% of one of the distances, 500px of "Background" on the side that is 1000px long, same with the other side. 

This gives you a buffer to position the map nicely on the screen but doesn't create an invite canvas where you would lose yourself. This would help with issue #67 as well  I guess.

**Comment by Kruptein** [comment]
Yes I think there should be some flexibility in the configuration and not just a binary infinite/finite.  A certain percentage like you mention could definitely be added.

I personally do like the infinite canvas, so I'm still planning on adding the beacon things I mentioned in #67.

P.S. The reason development might seem to have slown down recently is due to some big overhauls I'm doing.  The save/database rework was a big one, and I'm doing another one (hopefully the last one for a while) right now.

**Comment by schemen** [comment]
Haha don't worry :)
